Add notes about double forging - Closes #403 (#433)
LiskHQ/lisk-docs: modules/ROOT/pages/setup/binary.adoc

The installation may take a few minutes.
Check the output of the script to verify that the installation was successful.

> If you recognize an error, try to resolve it by analyzing the error output, otherwise, you can have a look at our xref:{url_dev_forum}[Dev Forum].

Reviewer: ```suggestion
If you recognize an error, try to resolve it by analyzing the error output, otherwise, please see the xref:{url_dev_forum}[Dev Forum].
```
